modules/ossm-about-sidecar-injection.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-sidecar-injection_{context}"]
+= About sidecar injection
+
+Sidecar injection is enabled via labels at the namespace or pod level. These labels also indicate the specific control plane managing the proxy. When you apply a valid injection label to the pod template defined in a deployment, any new pods created by that deployment automatically receive a sidecar. Similarly, applying a pod injection label at the namespace level ensures any new pods in that namespace include a sidecar.
+
+[NOTE]
+====
+Injection happens at pod creation through an admission controller, so changes appear on individual pods rather than the deployment resources. To confirm sidecar injection, check the pod details directly (using `oc describe`), where you can see the injected {istio} proxy container.
```

Comment:
I don't think that you need the parentheses here. I'd just go with "...using the 'oc describe' command, ..."